repository_service_tuf_worker/repository.py

```diff
@@ -1192,8 +1263,33 @@ def metadata_update(
             raise KeyError("No 'metadata' in the payload")
 
         if Root.type in metadata:
+            signing = self._settings.get_fresh(f"{Root.type.upper()}_SIGNING")
+            if signing:
+                root_signing = Metadata.from_dict(signing)
+                raise RepositoryError(
+                    "Root update process to version "
+                    f"{root_signing.signed.version} is waiting signature(s)"
+                )
+
             new_root = Metadata.from_dict(metadata[Root.type])
-            return self._root_metadata_update(new_root)
+            try:
+                return self._root_metadata_update(new_root)
+            except UnsignedMetadataError:
+                message = (
+                    "Metadata Update doesn't support Distributed Asynchronous "
+                    "Signing. Issue https://github.com/repository-service-tuf/"
+                    "repository-service-tuf-worker/issues/336"
+                )
```

Comment:
We don't know if "distributed asynchronous signing" was the intention here. Could be that the metadata had insufficient signatures for other reasons. Why not just let the `UnsignedMetadataError` from before fall through?